### ISSUE TITLE
Fail to build book

### ISSUE THREAD

**Issue by hadley** [Tazinho/Advanced-R-Solutions#142]
Any idea what the problem might be?

```
output file: 2-26-Deprecated_exercises.knit.md

Backtrace:
     █
  1. └─rmarkdown::render_site(encoding = "UTF-8")
  2.   └─generator$render(...)
  3.     ├─xfun::in_dir(...)
  4.     └─bookdown:::render_book_script(output_format, envir, quiet)
  5.       └─bookdown::render_book(...)
  6.         └─bookdown:::render_new_session(...)
  7.           └─bookdown:::merge_chapters(unlist(meta), main, orig = files)
  8.             ├─base::unlist(...)
  9.             └─base::mapply(...)
 10.               └─(function (f, o) ...
 11.                 └─xfun::read_utf8(f)
 12.                   └─base::readLines(con, encoding = "UTF-8", warn = FALSE)
 13.                     └─base::file(con, "r")
Error in file(con, "r") : cannot open the connection
Calls: <Anonymous> ... mapply -> <Anonymous> -> read_utf8 -> readLines -> file
In addition: Warning message:
In file(con, "r") :
  cannot open file '2-10-Function_factories.md': No such file or directory
Warning messages:
1: In file.rename(unlist(meta)[move], files_md[move]) :
  cannot rename file '2-10-Function_factories.md' to '_book/2-10-Function_factories.md', reason 'No such file or directory'
2: In file.rename(unlist(meta)[move], files_md[move]) :
  cannot rename file '2-24-Improving_performance.md' to '_book/2-24-Improving_performance.md', reason 'No such file or directory'
3: In file.rename(unlist(meta)[move], files_md[move]) :
  cannot rename file '1-13-Performance.md' to '_book/1-13-Performance.md', reason 'No such file or directory'
4: In file.rename(unlist(meta)[move], files_md[move]) :
  cannot rename file '1-14-Profiling.md' to '_book/1-14-Profiling.md', reason 'No such file or directory'
```

**Comment by hadley** [comment]
Oh it seems like `2-10-Function_factories.md` needs to attach purrr or something earlier so that the `%>%` in "Why might you formulate the Box-Cox transformation like this?" will work.

**Comment by henningsway** [comment]
Let me look into this shortly.

**Comment by henningsway** [comment]
closed by #141
